lib/stdlib/doc/src/edlin.xml

```diff
@@ -35,6 +35,33 @@
     and outputs action requests. The action requests are handled either by modules <c>group</c> or the <c>user_drv</c>.
     </p>
   </description>
+  <section>
+    <title>Key configuration</title>
+    <p>You can setup a custom key configuration that overrides the default key configuration.
+    This is done by specifying a config <c>-config kernel_keymap.config</c> when starting the shell.
+    The default key configuration is defined in the keymap found in <c>edlin_key</c> module.
+    If a custom keymap or keymap file is specified, then it will be merged with the default keymap.
```

Comment:
```suggestion
    <p>You can setup a custom key configuration that overrides the default key configuration.
    This is done by setting the stdlib application parameter <seeapp marker="stdlib_app#shell_keymap"><c>shell_keymap</c></seeapp> before Erlang is started.
    If you want to have the same keymap in all Erlang shells you can do so by putting a
    <seefile marker="config">config</seefile> file in your user's home directory and then set
    <seecom marker="erts:erl_cmd#ERL_FLAGS">ERL_FLAGS</seecom> to load it at startup.
    For example:
    </p>
    <pre>
$ cat $HOME/.erlang_keymap.config
[{stdlib,
  [{shell_keymap,
    #{ normal => #{ "\^[A" => clear } }
  }]
}].
$ ERL_FLAGS="-config $HOME/.erlang_keymap" erl
</pre>
    <p>The current keymap configuration can be fetched through <seeerl marker="#keymap/0">edlin:keymap()</seeerl>.
    If a custom keymap or keymap file is specified, then it will be merged with the default keymap.
```

I would suggest writing something like the above.

Reply:
I was thinking in the future, that we would want to move this configuration closer to user_drv, since we in theory would like to be able to set a custom keymap for the functions that happen on ctrl+c, ctrl+z, ctrl+g and (ctrl+d not implemented).

Reply:
I still think that we should have the config in stdlib as that is where I've put everything new we added in 26, no matter where the actual implementation is located. The shell_history things are in kernel for historical reasons, if I could move that to stdlib as well I would.